node/nodeHelper.go

```diff
@@ -212,6 +224,7 @@ func CreateNode(
 		WithPublicKeySize(config.ValidatorPubkeyConverter.Length),
 		WithNodeStopChannel(coreComponents.ChanStopNodeProcess()),
 		WithImportMode(isInImportMode),
+		WithESDTNFTStorageHandler(esdtNftStorage),
```

Comment:
as a next PR, I would drop the usage of the With... options and in this process we will remove all nil checks in the node's functions which simply clutter the functions and make them sub-optimal

Reply:
debatable, but agree about sub-optimal nil checks